airbyte-protocol/models/src/main/resources/protocol_models/airbyte_types.yaml

```diff
@@ -0,0 +1,11 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/models/src/main/resources/protocol_models/airbyte_types.yaml
+title: AirbyteType
+description: Airbyte supported types
+type: string
+enum:
+  - boolean
+  - number
+  - text
+  - object
```

Comment:
and we are saying array is just called object, right?

Reply:
yes